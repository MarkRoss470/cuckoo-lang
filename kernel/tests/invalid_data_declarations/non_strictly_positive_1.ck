data Invalid : Prop where
  | c : (Invalid -> Type) -> Invalid