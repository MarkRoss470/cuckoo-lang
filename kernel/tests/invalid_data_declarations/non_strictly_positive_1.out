ERROR at 2:10-2:17: Invalid cannot be referenced here. The inductive datatype being constructed can only be referenced in strictly positive positions.